go/consensus/genesis/genesis.go

```diff
@@ -5,13 +5,15 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature"
 	"github.com/oasislabs/oasis-core/go/consensus/api/transaction"
 )
 
 // Genesis contains various consensus config flags that should be part of the genesis state.
 type Genesis struct {
-	Backend    string     `json:"backend"`
-	Parameters Parameters `json:"params"`
+	Backend            string                `json:"backend"`
+	Parameters         Parameters            `json:"params"`
+	PublicKeyBlacklist []signature.PublicKey `json:"public_key_blacklist,omitempty"`
```

Comment:
Will we want to change the blacklist anytime in the future (e.g., via an unspecified governance mechanism)? If yes, then we may want to have this under `Parameters` instead.